packages/terra-consumer-icon/scripts/src/generate-example/generateStaticIcons.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable */
```

Comment:
wow, with this much the same as terra-icon is there no way to re-use packages or something? I haven't dug into terra-icon yet, but this seems like alot of copy paste